llvm/docs/LangRef.rst

```diff
@@ -12389,6 +12390,8 @@ Example:
       %X = fpext float 3.125 to double         ; yields double:3.125000e+00
       %Y = fpext double %X to fp128            ; yields fp128:0xL00000000000000004000900000000000
 
+.. _i_fptoui:
```

Comment:
Why sitofp but not uitofp? 

Can you add some notes on the interpretation of these? I assume afn is the only meaningful one?